docet-core/pom.xml

```diff
@@ -93,5 +93,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <!-- usiamo solo netty buffer -->
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
```

Comment:
I think we only need netty-buffer for this project, on downstream project which use netty-all it won't be a problem, but for Docet it is a too big dependency

Reply:
good, makes sense.